clang/test/SemaCXX/ctad-decay.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -Wno-unused-value -std=c++20 %s
+// expected-no-diagnostics
```

Comment:
can you move that test to clang/test/SemaCXX/cxx1z-class-template-argument-deduction.cpp ?

Reply:
Sure, done.